.github/workflows/ci.yml

```
@@ -147,6 +165,175 @@ jobs:
     if: always()
     
     steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)